packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -727,24 +729,16 @@ class VmServiceObjectLink<T> extends StatelessWidget {
         text = code.name!;
       } else if (object is InstanceRef) {
         final instance = object as InstanceRef;
-        if (instance.kind == InstanceKind.kTypeParameter) {
```

Comment:
do we no longer need to support this display for kTypeParameter? looks like this will hit the else case now

Reply:
This was just wrong and causing a crash.